yarn.lock

```diff
@@ -2478,10 +2478,10 @@
     "@jridgewell/resolve-uri" "^3.0.3"
     "@jridgewell/sourcemap-codec" "^1.4.10"
 
-"@koa/cors@3.4.2":
-  version "3.4.2"
-  resolved "https://registry.yarnpkg.com/@koa/cors/-/cors-3.4.2.tgz#70c13e5843d1762ce78fd8767162cd916132c946"
-  integrity sha512-NJU7/+h9XAfw/W/dLadDg8JYrQ5EDxstBl9a9G0A++EqvrQpabWcZ4tBxOdW57QjketX66zkOcXE+5V7IjLWYA==
+"@koa/cors@3.4.3":
+  version "3.4.3"
+  resolved "https://registry.yarnpkg.com/@koa/cors/-/cors-3.4.3.tgz#d669ee6e8d6e4f0ec4a7a7b0a17e7a3ed3752ebb"
+  integrity sha512-WPXQUaAeAMVaLTEFpoq3T2O1C+FstkjJnDQqy95Ck1UdILajsRhu6mhJ8H2f4NFPRBoCNN+qywTJfq/gGki5mw==
```

Comment:
Mh, two questions:

1. Why does git think this differs from the `main` branch?
2. Why wasn't the `yarn.lock` file not updated in https://github.com/strapi/strapi/pull/14677 ?

//cc @Convly

Reply:
Great catch. I assume no because the lockfile was not updated in that PR? But my lockfile is incorrect because we want it at the version from @Convly's PR...

Reply:
Yeah, good catch indeed, no idea why it has not modified the yarn lock when running the yarn setup (either manually or when running it for the publish ops)

Reply:
Do you want to amend it in a different PR or should I amend here?

Reply:
I think it's ok to do it here and then merge it asap.

Reply:
Looking at `main` we want `3.4.3` so this is correct.